src/Components/LandingPage/LandingPage.tsx

```diff
@@ -168,36 +180,65 @@ const LandingPage = ({ handleCheckboxChange }: LandingPageProps) => {
         </CardContent>
       </Card>
       <Box sx={{ mt: '.5rem' }}>
-        <Box>
-          <FormControlLabel
-            control={
-              <Checkbox
-                checked={formData.agreeToTermsOfService}
-                onChange={handleCheckboxChange}
-                sx={privacyErrorController.showError ? { color: '#c6252b' } : {}}
-              />
-            }
-            label={createCheckboxLabel()}
-            value="agreeToTermsOfService"
-          />
-          {privacyErrorController.showError && privacyErrorController.message(null)}
-        </Box>
-        <Box sx={{ mt: '.5rem' }}>
-          <FormControlLabel
-            control={
-              <Checkbox
-                checked={formData.is13OrOlder}
-                onChange={handleCheckboxChange}
-                sx={ageErrorController.showError ? { color: '#c6252b' } : {}}
+        <Card variant="outlined" sx={{ backgroundColor: '#FBF9FC' }}>
+          <CardContent>
+            <FormControlLabel
+              control={
+                <Checkbox
+                  checked={formData.agreeToTermsOfService}
+                  onChange={handleCheckboxChange}
+                  sx={
+                    privacyErrorController.showError
+                      ? { color: '#c6252b' }
+                      : formData.agreeToTermsOfService
+                      ? {
+                          color: '#41528C',
+                          '&.Mui-checked': {
+                            color: '#41528C',
+                          },
+                        }
+                      : {}
+                  }
```

Comment:
I think that updating the `styleController` will remove the need to specify the color here.